pkg/agent/agent.go

```diff
@@ -31,7 +31,8 @@ import (
 )
 
 const (
-	timeout                = 10 * time.Second
+	InitialTimeout         = 30 * time.Second
```

Comment:
With gRPC backoff this would amount to: 1 + 1.6 + 2.5 + 4 +6.5 + 10.5 -> CRASH after 30s, last attempt at 26.1s.